src/components/back-to-top/example-back-to-top.njk

```diff
@@ -0,0 +1,37 @@
+        </div>
+    </div>
+	{{
```

Comment:
Some of this file uses tabs to indent, to keep it consistent we use spaces because tabs can be different widths depending on how they are rendered but spaces are always the same size. This way we can keep the indenting consistent. This line here is one example where I know the format looks correct in vscode but here it looks off because the tab has been rendered as 8 spaces. Id recommend checking "insert spaces when pressing tab" in your vscode settings